package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "ember-bootstrap",
-  "version": "0.11.3-bayzat.8",
+  "version": "0.11.3-bayzat.9",
```

Comment:
Please don't bump version here, I'll do it separately including a proper tag… see the history of our branch: https://github.com/bayzat/ember-bootstrap/commits/v0.11.x-bayzat